agent/pkg/dagent/utils/container.go

```diff
@@ -156,7 +156,13 @@ func ExecTraefik(ctx context.Context, traefikDeployReq model.TraefikDeployReques
 		_ = removeContainer("traefik")
 	}
 
-	builder := containerbuilder.NewDockerBuilder(ctx).WithImage("index.docker.io/library/traefik:v2.6").
+	err = CreateNetwork("traefik", "bridge")
+	if err != nil {
+		log.Println("create traefik network error: " + err.Error())
+		return err
+	}
+
+	builder := containerbuilder.NewDockerBuilder(ctx).WithImage("index.docker.io/library/traefik:latest").
```

Comment:
Latest tags can contain breaking changes. Are we willing to support that, and dedicate time to occasional breakages?

Reply:
Good point, I'll change to a fix version.